src/DotNetBridge/NativeDataView.cs

```diff
@@ -357,8 +355,7 @@ protected override bool MoveNextCore()
                     return index < _view._rowCount;
                 }
 
-                public static IRowCursor[] CreateSet(out IRowCursorConsolidator consolidator,
-                IChannelProvider provider, NativeDataView view, bool[] active, int n, IRandom rand)
+                public static RowCursor[] CreateSet(IChannelProvider provider, NativeDataView view, bool[] active, int n, Random rand)
```

Comment:
>CreateSet [](start = 42, length = 9)

If you are only creating cursors using this method, the constructor can be private.

Reply:
made the ctor private

---
In reply to: [248382326](https://github.com/Microsoft/NimbusML/pull/77#discussion_r248382326) [](ancestors = 248382326)